docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,441 @@
+
+```
+Developer Certificate of Origin
+Version 1.1
+
+Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+
+Everyone is permitted to copy and distribute verbatim copies of this
+license document, but changing it is not allowed.
+
+
+Developer's Certificate of Origin 1.1
+
+By making a contribution to this project, I certify that:
+
+(a) The contribution was created in whole or in part by me and I
+    have the right to submit it under the open source license
+    indicated in the file; or
+
+(b) The contribution is based upon previous work that, to the best
+    of my knowledge, is covered under an appropriate open source
+    license and I have the right under that license to submit that
+    work with modifications, whether created in whole or in part
+    by me, under the same open source license (unless I am
+    permitted to submit under a different license), as indicated
+    in the file; or
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
+
+#### DCO Sign-Off Methods
+
+The DCO requires a sign-off message in the following format to appear on each commit in the pull request:
+
+```
+Signed-off-by: Humpty Dumpty <humpty.dumpty@example.com>
+```
+
+The DCO text can either be manually added to your commit body, or you can add either **-s** or **--signoff** to your usual git commit commands. If you are using the GitHub UI to make a change you can add the sign-off message directly to the commit message when creating the pull request. If you forget to add the sign-off you can also amend a previous commit with the sign-off by running **git commit --amend -s**. If you've pushed your changes to GitHub already you'll need to force push your branch after this with **git push -f**.
+
+**ATTRIBUTION**:
+
+- https://probot.github.io/apps/dco/
+- https://github.com/opensearch-project/common-utils/blob/main/CONTRIBUTING.md
+- https://code.asam.net/simulation/wiki/-/wikis/docs/project_guidelines/ASAM-DCO?version_id=c510bffb1195dc04deb9db9451112669073f0ba5
+- https://thesofproject.github.io/latest/contribute/contribute_guidelines.html
+
+## Source file structure [Preliminary]
+
+The source files for the [Keptn Lifecycle Toolkit](https://lifecycle.keptn.sh/docs) are stored under the *docs/content/en/docs* directory in the repository.
+The build strategy is to build everything except for:
+
+* The files/folders that are listed in the `ignoreFiles` array in the [config.toml](https://github.com/keptn/keptn.github.io/blob/master/config.toml) file
```

Comment:
I think this is not necessary for the lifecycle-toolkit docs as the versions are appended automatically and the config file only exists in the target repo

Reply:
So where is the list of files for git to ignore -- like .*.sw?